<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Deactivated Data Engine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background:#ffffff
}</string>
  </property>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>460</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ffffff</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_load">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>260</y>
     <width>490</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	border:none;
	text-align:center;
}
QProgressBar::chunk
{
	background-color:#8cc540;
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_msg">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>240</y>
     <width>210</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Checking server connection ...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
